watson/utils.py

```diff
@@ -71,16 +91,11 @@ def _style_tags(tags):
     def _style_short_id(id):
         return style('id', id[:7])
 
-    formats = {
-        'project': {'fg': 'magenta'},
+    formats = DEFAULT_STYLES.copy()
+    formats.update({
         'tags': _style_tags,
-        'tag': {'fg': 'blue'},
-        'time': {'fg': 'green'},
-        'error': {'fg': 'red'},
-        'date': {'fg': 'cyan'},
         'short_id': _style_short_id,
-        'id': {'fg': 'white'}
-    }
+    })
```

Comment:
I think we should extract the nested functions & add them in `get_styles_from_config` in order to avoid a copy every time we call `style` :) .

Reply:
@k4nar I think I need your review again. I solved the performance problem that you mentioned but I followed a slightly different way.